.github/workflows/workflow.yml

```diff
@@ -78,3 +80,23 @@ jobs:
         with:
           name: artifacts
           path: otel-dotnet-autoinstrumentation-macos.tar
+
+  create_release:
+    name: Create release
+    runs-on: ubuntu-20.04
+    if: startsWith(github.ref, 'refs/tags/') # only trigger if the current event is for a tag push
+    needs: [alpine_build, linux_build, windows_build, macos_build]
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          path: .
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

Comment:
I do not trust `softprops/action-gh-release` especially that it is automatically bumped. Use https://cli.github.com/manual/gh_release_create instead. The tool is already installed.